### ISSUE TITLE
no support centos?

### ISSUE THREAD

**Issue by awakeljw** [archibate/vimrc#2]
# github.com/junegunn/fzf/src/tui
src/tui/tcell.go:18:6: HasFullscreenRenderer redeclared in this block
	previous declaration at src/tui/dummy.go:7:30
src/tui/tcell.go:27:6: Attr redeclared in this block
	previous declaration at src/tui/dummy.go:5:6
src/tui/tcell.go:75:2: Bold redeclared in this block
	previous declaration at src/tui/dummy.go:20:19
src/tui/tcell.go:76:2: Dim redeclared in this block
	previous declaration at src/tui/dummy.go:21:24
src/tui/tcell.go:77:2: Blink redeclared in this block
	previous declaration at src/tui/dummy.go:24:24
src/tui/tcell.go:78:2: Reverse redeclared in this block
	previous declaration at src/tui/dummy.go:26:24
src/tui/tcell.go:79:2: Underline redeclared in this block
	previous declaration at src/tui/dummy.go:23:24
src/tui/tcell.go:80:2: Italic redeclared in this block
	previous declaration at src/tui/dummy.go:22:24
src/tui/tcell.go:84:2: AttrUndefined redeclared in this block
	previous declaration at src/tui/dummy.go:16:23
src/tui/tcell.go:85:2: AttrRegular redeclared in this block
	previous declaration at src/tui/dummy.go:17:28
src/tui/tcell.go:85:2: too many errors
make: *** [Makefile:130: target/fzf-linux_amd64] Error 2

**Comment by archibate** [comment]
Yes, we only support Arch, Ubuntu, MacOS for now. For other distro the script will build from source. In this case fzf failed to build due to a very low go version. Maybe need install a new go from source too?
If you figured out how to fix the go version on CentOS, you may open a pull request to fix, like @RakerZh did in https://github.com/archibate/vimrc/pull/1 for fixing MacOS.

**Comment by RakerZh** [comment]
try to download go from source.

I think maybe something like this implemented in `install.sh`
`    if [ "$(uname -m)" == "x86_64" ]; then
        wget https://go.dev/dl/go1.18.linux-amd64.tar.gz
        sudo tar -C /usr/local -xzf go1.18.linux-amd64.tar.gz

        if [ -n "$BASH_VERSION" ]; then
          echo "export PATH=$PATH:/usr/local/go/bin" >> ~/.bashrc
          source ~/.bashrc
        elif [ -n "$ZSH_VERSION" ]; then
          echo "export PATH=$PATH:/usr/local/go/bin" >> ~/.zshrc
          source ~/.zshrc
        else
          echo "add go to your PATH! "
        fi

    elif [ "$(uname -m)" == "aarch64" ]; then
        wget https://go.dev/dl/go1.18.linux-arm64.tar.gz
        sudo tar -C /usr/local -xzf go1.18.linux-arm64.tar.gz

        if [ -n "$BASH_VERSION" ]; then
          echo "export PATH=$PATH:/usr/local/go/bin" >> ~/.bashrc
          source ~/.bashrc
        elif [ -n "$ZSH_VERSION" ]; then
          echo "export PATH=$PATH:/usr/local/go/bin" >> ~/.zshrc
          source ~/.zshrc
        else
          echo "add go to your PATH! "
        fi

    else
        echo "golang is not supported on this architecture"
    fi`

**Comment by xxy-im** [comment]
#6
